posix/unix.c

```diff
@@ -53,7 +58,15 @@ typedef struct _unixsock_t {
 
 	spinlock_t spinlock;
 
+	/* Socket this one is connected to */
+	struct _unixsock_t *remote;
+
+	/*
+	 * For SOCK_DGRAM: list of sockets connected to this socket.
+	 * For other types: list of sockets requesting a connection.
+	 */
 	struct _unixsock_t *connect;
```

Comment:
Wondering about a better name for this. It was already misleading before the change and now it is even worse as `connect` may wrongly suggest something about the the state of sockets stored while the state varies across socket types (connected vs. requesting a connection). Maybe some general name (`queue`, whatever) would be better, to emphasize that the list serves multiple purposes?